pom.xml

```diff
@@ -113,6 +113,12 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
```

Comment:
It's up to your preferences, but I feel we already have a few testing frameworks like `junit` and `truth`, which should be good enough. Is there a specific feature that only `assertj` can do?

Reply:
`assertj` is required for OpenTelemetry assertions. I've attempted to remove this package, but the tests are unable to compile without it.